middleman-core/lib/middleman-core/core_extensions.rb

```diff
@@ -100,10 +100,10 @@
   Middleman::Extensions::MinifyCss
 end
 
-# MinifyJavascript compresses JS
+# MinifyJavaScript compresses JS
 Middleman::Extensions.register :minify_javascript do
   require 'middleman-core/extensions/minify_javascript'
-  Middleman::Extensions::MinifyJavascript
+  Middleman::Extensions::MinifyJavaScript
```

Comment:
Note: this is probably an api break, I can drop it fairly easily.

Reply:
I think this would only be an internal API break, would be safe to apply in all places (the actually extension class). `:minify_javascript` is the public name